execution/execution_test.go

```diff
@@ -0,0 +1,174 @@
+package execution
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/mesg-foundation/core/service"
+)
+
+var (
+	srv = &service.Service{
+		Tasks: []*service.Task{
+			&service.Task{
+				Key: "task",
+				Inputs: []*service.Parameter{
+					&service.Parameter{Key: "foo", Type: "String"},
+					&service.Parameter{Key: "bar", Type: "String"},
+				},
+				Outputs: []*service.Output{
+					&service.Output{
+						Key: "outputX",
+						Data: []*service.Parameter{
+							&service.Parameter{
+								Key:  "foo",
+								Type: "String",
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	taskKey       = "task"
+	defaultInputs = map[string]interface{}{
+		"foo": "hello",
+		"bar": "world",
+	}
+	tags = []string{"tag1", "tag2"}
+)
+
+func db(t *testing.T, dir string) DB {
+	fmt.Println(dir)
+	db, err := New(dir)
+	require.NoError(t, err)
+	return db
+}
+
+func TestCreate(t *testing.T) {
+	db := db(t, filepath.Join(os.TempDir(), "TestCreate"))
```

Comment:
- Files are not deleted after tests
- use https://golang.org/pkg/io/ioutil/#TempFile